asistente_ladm_col/logic/ladm_col/pg_ladm_query.py

```diff
@@ -213,7 +220,11 @@ def get_invalid_col_party_type_no_natural(db):
                            lc_party_type_d=db.names.LC_PARTY_TYPE_D,
                            lc_party_document_type_d=db.names.LC_PARTY_DOCUMENT_TYPE_D,
                            lc_party_document_type_d_ilicode_f_nit_v=LADMNames.LC_PARTY_DOCUMENT_TYPE_D_ILICODE_F_NIT_V,
-                           lc_party_type_d_ilicode_f_not_natural_party_v=LADMNames.LC_PARTY_TYPE_D_ILICODE_F_NOT_NATURAL_PARTY_V)
+                           lc_party_document_type_d_ilicode_f_sequential_v=LADMNames.LC_PARTY_DOCUMENT_TYPE_D_ILICODE_F_SEQUENTIAL_V,
+                           lc_party_type_d_ilicode_f_not_natural_party_v=LADMNames.LC_PARTY_TYPE_D_ILICODE_F_NOT_NATURAL_PARTY_V,
+                           lc_party_t_first_name_2_f=db.names.LC_PARTY_T_FIRST_NAME_2_F,
+                           lc_party_t_surname_2_f=db.names.LC_PARTY_T_SURNAME_2_F,
+                           lc_party_t_genre_f=db.names.LC_PARTY_T_GENRE_F)
```

Comment:
Just a question, did you take into consideration the marital status, considering that it was added to version 1.2 of the model?

Reply:
The marital status is not validated because in the quality rules defined by the IGAC this field is not validated and in the cadastral survey model the field is optional.

Reply:
But in this case a legal party cannot have a marital status, so I think we should validate it

Reply:
change made in commit ce34c93